QLogger.h

```diff
@@ -75,13 +80,15 @@ class QLoggerManager
     * @param level The maximum level allowed.
     * @param fileFolderDestination The complete folder destination.
     * @param mode The logging mode.
-    * @param fileSuffixIfFull The filename suffix if the file is full.
+    * @param fileTag The file tag.
+    * @param fileHandling file handling modes
     * @param messageOptions Specifies what elements are displayed in one line of log message.
     * @return Returns true if any error have been done.
     */
    bool addDestination(const QString &fileDest, const QStringList &modules, LogLevel level = LogLevel::Warning,
-                       const QString &fileFolderDestination = QString(), LogMode mode = LogMode::OnlyFile,
-                       LogFileDisplay fileSuffixIfFull = LogFileDisplay::DateTime,
+                       const QString &fileFolderDestination = QString(), LogMode mode = LogMode::Default,
+                       LogFileTag fileTag = LogFileTag::Default,
+                       LogFileHandling fileHandling = LogFileHandling::Default,  
```

Comment:
This causes an unnecessary breaking change right now. I'd ask you to add a new method or add the parameter at the end of the declaration.

Reply:
I can put it just before the notify parameter if it is not already there, because putting it after the notify parameter is messy in my opinion and does not make much sense to me.

Reply:
After a second thought, just leave it there. It's a breaking change but makes sense since you're configuring all "file" options one after the other. In the future I might modify that to create a "file config" and "log config".